tests/api_fastapi/core_api/routes/public/test_connections.py

```diff
@@ -796,3 +796,10 @@ def test_should_respond_403_by_default(self, test_client, body):
             "detail": "Testing connections is disabled in Airflow configuration. "
             "Contact your deployment admin to enable it."
         }
+
+
+class TestCreateDefaultConnections(TestConnectionEndpoint):
+    def test_should_respond_204(self, test_client):
+        response = test_client.post("/public/connections/defaults")
+        assert response.status_code == 204
+        assert response.content == b""
```

Comment:
Maybe mock `db_create_default_connections` and assert it has been called 1 time. Just to be sure that the endpoint is actually doing something.

Reply:
Nice point, included. Thanks!